swift/ql/test/library-tests/dataflow/dataflow/test.swift

```diff
@@ -284,13 +284,13 @@ func testTuples() {
 
     sink(arg: t1)
     sink(arg: t1.0)
-    sink(arg: t1.1) // $ flow=277
+    sink(arg: t1.1)
```

Comment:
Yay for https://github.com/github/codeql/blob/main/swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll#L527 🎉